src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -72,108 +81,156 @@ export default function (AB) {
             id: ids.component,
             resize: true,
             modal: true,
-            height: 500,
-            width: 700,
+            boarderless: true,
+            height: 503,
+            width: 692,
             head: {
                view: "toolbar",
                css: "webix_dark",
-               cols: [
+               paddingX: 2,
+               elements: [
                   {
                      view: "label",
-                     label: L("Add new field"),
-                     css: "modal_title",
+                     id: ids.title,
                      align: "center",
+                     label: L("<span>Choose Field-Type</span>"),
```

Comment:
Do we need the <span> tag or can it be removed?

Reply:
label: L("Choose Field-Type"),
label: "<span>" + L("Chooose Field-Type") + "</span>",